ReadingRainbow/Models/Recording.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Recording.swift
+//  ReadingRainbow
+//
+//  Created by Ami Smith on 8/19/23.
+//
+
+import Foundation
+
+struct Recording: Hashable, Codable, Identifiable {
+    var id = UUID()
```

Comment:
This can probably be a constant (let)

Reply:
I tried changing it to a let and got this error: immutable property will not be decoded because it is declared with an initial value which cannot be overwritten.
I'm not super sure what this means...

Reply:
This means you'll have to write an initializer for this object. Xcode should auto complete one for you.
Then you'll move the `= UUID()` to inside the init and have the constant be declared like `let id: UUID`.